test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("[Docker] [Kubeadm] - [management.cattle.io/v3] Create and dele
 	})
 })
 
-var _ = Describe("[Docker] [RKE2] - [management.cattle.io/v3] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.DontRunLabel), func() {
+var _ = Describe("[Docker] [RKE2] - [management.cattle.io/v3] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.FullTestLabel), func() {
```

Comment:
as pointed out by @salasberryfin, how about enabling RKE2 tests on full tests since it is main target and run Azure in short tests https://github.com/rancher/turtles/blob/4e377f956d2e0ecf610ba2f779c1544daca8fedc/test/e2e/suites/v2prov/v2prov_test.go#L41 instead?

Reply:
I don’t have the full context of discussion, it is worth explaining this once again. I believe that
> how about enabling RKE2 tests on full tests

is already solved by this line.

As for the second part, this is out of scope for this PR.